docs/developer-guide/toolchain-guide.md

```diff
@@ -336,6 +336,8 @@ The first thing you can do to test whether your build toolchain is setup correct
 
 This should return without any error.
 
+If `codegen-local` returns the error `Could not make proto path relative` with a file path under `~/go/src/`, this may mean that your local Argo CD repository is cloned at an unsupported path. Please refer to https://argo-cd.readthedocs.io/en/stable/developer-guide/toolchain-guide/#clone-the-argo-cd-repository-from-your-personal-fork-on-github  
+
```

Comment:
From what I recall, I did a fix so Argo could be cloned in any repository and not only GO_PATH.

Reply:
It didn't work for codegen-local, most of the other make local targets I tried work when the repo is nit under $GOPATH. If this was intended to be supported,  I will take a deeper look in the code, as this might be a bug then.
But the docs before my change state that the assumption is that the repo is under ~/go/src, so I either fix the bug and remove this requirement from the docs entirely or my suggested change is still relevant.